frontend/src/main/pages/AdminJobsPage.js

```diff
@@ -71,16 +57,24 @@ const AdminJobsPage = () => {
                         <TestJobForm submitAction={submitTestJob} />
                     </Accordion.Body>
                 </Accordion.Item>
-                {
-                    jobLaunchers.map((jobLauncher, index) => (
-                        <Accordion.Item eventKey={index + 1}>
-                            <Accordion.Header>{jobLauncher.name}</Accordion.Header>
-                            <Accordion.Body>
-                                {jobLauncher.form}
-                            </Accordion.Body>
-                        </Accordion.Item>
-                    ))
-                }
+                <Accordion.Item eventKey="1">
+                   <Accordion.Header>Milk The Cows</Accordion.Header>
+                   <Accordion.Body>
+                       <JobComingSoon/>
+                   </Accordion.Body>
+               </Accordion.Item>
+               <Accordion.Item eventKey="2">
+                   <Accordion.Header>Instructor Report</Accordion.Header>
+                   <Accordion.Body>
+                       <InstructorReportJobForm submitAction={submitTestJob} />
```

Comment:
So, the user is going to click on the InstructorReportJobForm, but end up submitting a test job?

I think the user will be very unsatisfied with that.  By calling `submitTestJob` you just submit another version of the test job, which makes this entire panel pretty useless.   

If you want to submit an InstructorReportJob, you have to actually submit an instructor report job.   That means you need  a whole different set of the functions on lines 15-46.

You can use this file from the courses project as a model:

https://github.com/ucsb-cs156-f22/f22-5pm-courses/blob/main/frontend/src/main/pages/AdminJobsPage.js